app/services/registration_service.rb

```diff
@@ -1,7 +1,10 @@
 class RegistrationService
   def from_invitation(email, invitation_token, user_data = {})
-    Invitation.find_by(email: email, token: invitation_token)
-    User.create({ email: email }.merge(user_data))
+    invitation = Invitation.find_by(email: email, token: invitation_token)
+    user = User.where(email: email).first
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1227578/8145760/bf34fe9a-1215-11e5-9084-8dbe72140ccd.png)

I'll check this PR later =)